boards/nxp/imx93_evk/doc/index.rst

```diff
@@ -85,6 +85,23 @@ Serial Port
 This board configuration uses a single serial communication channel with the
 CPU's UART4.
 
+Board MUX Control
+-----------------
+
+This board configuration uses a series of digital multiplexers to switch between
+different board functions. The multiplexers are controled by a GPIO signal called
+``EXP_SEL`` from onboard GPIO expander ADP5585. This signal is configured if a
+board function that requires configuring the mux is enabled.
```

Comment:
"EXP_SEL" can be configured to select function set "A" or "B" by dts configuration if board control moduel is enabled.